apps/picsa-tools/resources-tool/src/app/pages/resources/search.component.html

```diff
@@ -0,0 +1,33 @@
+<div>
+  <input
+    type="search"
+    name="filter"
+    class="form-control"
+    placeholder="Search resources...."
+    [(ngModel)]="query"
+    (input)="onSearchInputChange($event)"
+    autofocus
+  />
+  <div *ngIf="searchResults.length > 0" class="card">
+    <div class="card-content" *ngFor="let result of searchResults">
+      <!-- <mat-card>
```

Comment:
nit(non-blocking)
Did you manage to get the material components working in the end? If yes would be good to include here. If no you could look at importing the `MatCardModule` into the `search.component.ts` file imports section

Reply:
thank you, this will be helpful i will post any challenges i come across on slack